src/declarative-event-listeners.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {dedupingMixin} from '@polymer/polymer/lib/utils/mixin.js';
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+
+export interface SuperClassConstructor {
+  // tslint:disable-next-line:no-any Required for constructor signature.
+  new(...args: any[]): PolymerElement;
```

Comment:
does TS 3.0 support `...args: unknown[]` for mixins?

Reply:
Nope, it breaks. Also don't want to use `unknown` for now because of g3.